mkdocs.yml

```diff
@@ -33,8 +33,7 @@ nav:
               - Dashboard: components/admin-app/dashboard.md
           - Question-Answering Service:
               - components/qa-service/index.md
-              - Semantic Search: components/qa-service/semantic-search.md
-              - LLM Response: components/qa-service/llm-response.md
+              - Search: components/qa-service/search.md
```

Comment:
Is there a title that can better express that this can do both search AND LLM-generated responses?